test/Interop/Cxx/stdlib/Inputs/fake-toolchain/include/c++/v1/fake-toolchain.h

```diff
@@ -7,4 +7,8 @@ void foo(int x) {}
 
 }; // namespace FakeNamespace
 
+namespace std {
+typedef unsigned long size_t;
```

Comment:
will this get tested on windows? `size_t` needs unsigned long long for the windows target.

Reply:
This won't be tested on Windows, since Clang on Windows doesn't support `-stdlib=libc++` which is used for fake toolchain tests.
There was an effort to support it in Clang, however, the patch got reverted: https://reviews.llvm.org/D101479

Reply:
I see, great